frontend/packages/topology/src/components/svg/SvgCircledIcon.tsx

```diff
@@ -39,17 +40,31 @@ const CircledIcon: React.FC<SvgTypedIconProps> = (
         cy={y + iconHeight / 2}
         r={iconWidth / 2 + padding}
       />
-      <g ref={typedIconRef}>
-        <image
-          key={`image-${FILTER_ID}`}
-          x={x - iconWidth}
-          y={y}
-          width={width}
-          height={height}
-          xlinkHref={getImageForIconClass(iconClass) || iconClass}
-          filter={createSvgIdUrl(FILTER_ID)}
-        />
-      </g>
+      {icon ? (
+        <g ref={typedIconRef}>
+          <foreignObject
+            key={`image-${FILTER_ID}`}
+            x={-iconWidth}
```

Comment:
just verifying this change is needed right as we below https://github.com/openshift/console/pull/10721/files#diff-d1a68270878fe2df06bb4a025e09188cf6693875dae259d2f912f5702349a7adR59

```
 x={x - iconWidth}
```

Reply:
Did `x={x - iconWidth + 3}`
![Screenshot from 2021-12-30 15-23-59](https://user-images.githubusercontent.com/22490998/147741081-37547eb3-4ee5-45e1-8ef3-0ae4f366c072.png)